bin/run-caliper/run-caliper.py

```diff
@@ -0,0 +1,33 @@
+import json

```

Comment:
how did you test this?
you have to deploy the SUT and run it against it in order for this PR to be accepted.

Reply:
yes, i want to test in GCP too. but currently, for JSON file, I tested it by seeing the output file it generated. for running caliper, probably anyone can help me how is it being tested. or probably we can clone our project in the cloud instance?